src/parser/jobs/drg/modules/Drift.tsx

```diff
@@ -77,7 +77,10 @@ export default class Drift extends Module {
 
 		if (this.downtime.isDowntime(plannedUseTime)) {
 			const downtimeWindow = this.downtime.getDowntimeWindows(plannedUseTime)[0]
-			expectedUseTime = downtimeWindow.end
+
+			// there's a case where getDowntimeWindows will return null even though the planned use time is marked as downtime
+			// adding a fallback to ensure the parse is still viewable.
+			expectedUseTime = (downtimeWindow) ? downtimeWindow.end : plannedUseTime
```

Comment:
This can be simpler, and a little more accurate at only coalescing on null, rather than falsey (not that it should be falsey if it isn't null but 🤷) 
```suggestion
			expectedUseTime = downtimeWindow?.end ?? plannedUseTime
```

Reply:
works for me! wasn't aware of this syntax but I am now and I like it.